packages/snaps-simulator/src/features/renderer/components/Text.tsx

```diff
@@ -15,13 +15,26 @@ export const Text: FunctionComponent<TextProps> = ({ node, id }) => {
 
   return (
     <ReactMarkdown
-      allowedElements={['p', 'strong', 'em']}
+      allowedElements={['p', 'strong', 'em', 'a']}
       components={{
         p: ({ children: value }) => (
           <ChakraText fontFamily="custom" fontSize="sm" paddingBottom="1">
             {value}
           </ChakraText>
         ),
+        // TODO: Update styling
```

Comment:
Are you planning on doing that in this PR, or separate? If separate, please create an issue for it.

Reply:
Yup, I still need to see the design with @eriknson

Reply:
Ok, can you link the issue here and in the code?

Reply:
https://github.com/MetaMask/snaps/issues/1881